doc/api/esm.md

```diff
@@ -657,21 +668,24 @@ export async function resolve(specifier, context, defaultResolve) {
 }
 ```
 
-#### `getFormat(url, context, defaultGetFormat)`
+#### `load(url, context, defaultLoad)`
 
-> Note: The loaders API is being redesigned. This hook may disappear or its
-> signature may change. Do not rely on the API described below.
+> Note: Previously the experimental version had 3 separate hooks, which have
+> been consolidated into this one.
 
 * `url` {string}
 * `context` {Object}
-* `defaultGetFormat` {Function}
+  * `format` {string} The format optionally supplied by the `resolve` hook.
+  **Default:** `undefined`
```

Comment:
Ditto, I don’t know what ‘Default’ means here.

Reply:
It means the default value of `format` is undefined

Reply:
I agree using default is a bit confusing here, instead I think it should be:

```js
  * `format` {string|undefined} The format optionally supplied by the `resolve` hook.
```

Reply:
@aduh95 I think `null` is not a valid type for this though—either `string` or nothing (`undefined`).

Tangentially related: Is there a version of optional that node's linter/parser understands? It's almost universally a question mark, like `{string?}` (or sometimes `{?string}`), but the test seems to barf.

Reply:
> Is there a version of optional that node’s linter/parser understands? It’s almost universally a question mark

I don’t know offhand the answer to this, but in JSDoc comments you define an optional parameter by putting it in brackets: ``* `[format]` {string} The format...``. If the linter doesn’t accept that then I would just not worry about it, you already say in the description that it’s optional.